modules/system/applications.nix

```diff
@@ -37,11 +38,40 @@ in
         rm ~/Applications/'Nix Apps'
       fi
 
-      if [ ! -e '/Applications/Nix Apps' ] \
-         || ourLink '/Applications/Nix Apps'; then
-        ln -sfn ${cfg.build.applications}/Applications '/Applications/Nix Apps'
+      ourfolder () {
+        local marker
+        marker=$(xattr -p org.nix-darwin "$1" 2>/dev/null)
```

Comment:
`xattr -p org.nix-darwin '/Applications/Nix Apps'` returns 1 for me, but I guess that’s fine, since we want the function to fail in that case anyway.

(FWIW I don’t know if the complexity we do around detecting ownership is really worth it here; we don’t do it for fonts and probably nobody expects a directory called “Nix Apps” to be owned by anything other than nix-darwin? But I don’t mind keeping it for now, since probably the files interface will change how this works anyway.)

Reply:
I was thinking about getting rid of it. I mainly did it to stay in the spirit of the code above. I would be glad to get rid of this check.

Reply:
I did remove it, though not the existing code that checks for the symlink. What do you think @emilazy Should that also go?

Reply:
What existing code that checks for the symlink are you referring to? What's currently here looks fine